maraboupy/MarabouNetworkNNet.py

```diff
@@ -69,6 +209,10 @@ def __init__ (self, filename, normalize=False):
         # Set the number of variables
         self.numVars = self.numberOfVariables()
 
+    # def getMarabouQuery(self):
```

Comment:
Please remove commented-out code from the PR

Reply:
Done. Sorry, this method moved to the parent class, and I forgot to remove the commented out lines from here. 

Indeed, I have been having build issues with a new test that I've added (still figuring out how certain things work in pytest). Hopefully the last commit fixes the issue. Thanks for the comments!